config/prod.exs

```diff
@@ -46,8 +46,7 @@ config :appsignal, :config,
   active: true,
   name: System.get_env("APPSIGNAL_APP_NAME"),
   push_api_key: System.get_env("APPSIGNAL_API_KEY"),
-  env: Mix.env,
-  ignore_actions: ["ConsoleWeb.Router.DeviceController#show"]
+  env: Mix.env
```

Comment:
removing here since we want to keep track of this endpoint in the future in appsignal